kernel/kernel-api/src/main/java/io/delta/kernel/engine/ParquetReadResult.java

```diff
@@ -0,0 +1,22 @@
+package io.delta.kernel.engine;
+
+import io.delta.kernel.data.ColumnarBatch;
+
+public class ParquetReadResult {
+
+  private final ColumnarBatch data;
+  private final String filePath;
+
+  public ParquetReadResult(ColumnarBatch data, String filePath) {
+    this.data = data;
+    this.filePath = filePath;
+  }
+
+  public ColumnarBatch getData() {
+    return this.data;
```

Comment:
in java, we typically only say `this.` in the constructor. In this getter, it is very clear what `return data;` means